include/z64scene.h

```diff
@@ -334,11 +337,11 @@ typedef struct {
 
 typedef union {
     SCmdBase              base;
-    SCmdSpawnList         spawnList;
-    SCmdActorList         actorList;
+    SCmdPlayerEntryList   playerEntryList;
+    SCmdActorList         actorEntryList;
     SCmdUnused02          unused02;
     SCmdRoomList          roomList;
-    SCmdEntranceList      entranceList;
+    SCmdSpawnList         spawnList;
```

Comment:
Couple thoughts here.

Is there a reason to keep SCmdPlayerEntryList and SCmdActorList as separate types?

SCmdActorList -> SCmdActorEntryList

SCmdPlayerEntryList / SCmdActorList should be expressed as being the spawn data for actors, since they contain all of the parameters necessary to instantiate an actor instance. But I guess that goes back to the `Actor_SpawnEntry` naming discussion.

SCmdSpawnList I don't think is 100% the way there. Considering that player specific, it should represent that.